tests/integration/goldens/redis/setup.py

```diff
@@ -46,6 +46,12 @@
     "proto-plus >= 1.22.3, <2.0.0dev",
     "protobuf>=3.20.2,<6.0.0dev,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",
 ]
+extras = {
```

Comment:
Where do you use `extras`? From what I gather (eg https://xebia.com/blog/a-practical-guide-to-using-setup-py/, https://sukhbinder.wordpress.com/2023/04/09/a-short-primer-on-extra_requires-in-setup-py/) you need to pass this to `setup(..., extras_require = extras)`

Reply:
Good catch Victor! I missed setting extras_require  in setup. Fixed now.